modules/config.xqm

```diff
@@ -12,6 +12,10 @@ declare namespace system="http://exist-db.org/xquery/system";
 declare namespace expath="http://expath.org/ns/pkg";
 declare namespace repo="http://exist-db.org/xquery/repo";
 
+
+declare variable $config:login-domain := "org.exist.public-repo.login";
```

Comment:
How is this related to the PR?

Reply:
This will make more sense with the next commits

Reply:
$config:login-domain is now used in the controller and put-package.xq

Reply:
How about adding another variable here, rather than hardcoding `".user"` in those modules?